client/public/index.html

```diff
@@ -10,8 +10,8 @@
   <body>
     <div id="root"></div>
     <link
-      href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&amp;subset=latin-ext"
+      href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700&amp;subset=latin-ext"
```

Comment:
Montserrat font is from our landing page but unfortunately Roboto was more readable in my opinion, maybe we should increase the font size

Reply:
I think we will implement such changes when we gonna customize app layout, cause now it doesn't make sense. The whole app  gone look  different later on